tests/__snapshots__/SubMenu.spec.tsx.snap

```diff
@@ -83,7 +83,7 @@ HTMLCollection [
       </div>
       <div
         class="rc-menu-submenu rc-menu-submenu-popup custom-className rc-menu-submenu-placement-rightTop"
-        style="--arrow-x: 0px; --arrow-y: 0px; left: -1000vw; top: -1000vh; box-sizing: border-box;"
+        style="left: -1000vw; top: -1000vh; box-sizing: border-box;"
```

Comment:
<img width="1069" alt="图片" src="https://github.com/react-component/menu/assets/507615/cf9227b1-40b8-4cf1-841c-0863a3b3293a">

Reply:
Hmmm, I will revert the change on this - my bad but it could be better though. I mean it come from running update snapshot command on my machine and maybe caused by running on a newer node? should we upgrade CI's node version (which will happen on another PR of course)